### PR TITLE
fix(challenges): improved instructions for anchor element challenge

### LINKED ISSUE #16289: Improve instructions: Link to External Pages with Anchor Elements
#### Challenge Name
Basic HTML and HTML5: Link to External Pages with Anchor Elements
https://beta.freecodecamp.org/en/challenges/basic-html-and-html5/link-to-external-pages-with-anchor-elements

#### Issue Description
The image displaying the anchor elements is smashed together. I think it should be broken into smaller parts and displayed with regular text like below:

Let's break down the example:
```
<p>Here's` a <a href="http://freecodecamp.org"> link to freecodecamp.org</a> for you to follow.</p>
```
1. Normal text in the p element
```
<p>Here's a ... for you to follow.</p>
```
2. Anchor tag (requires a closing tag)
```
<a></a>
```
3. href is an anchor tag attribute
```
<a href="http://freecodecamp.org"></a>
```
4. The text within the anchor tags, `link to freecodecamp.org`, will display a link to click 
```
<p>Here's` a <a href="http://freecodecamp.org"> link to freecodecamp.org</a></p>
```
5. Final Render
<p>Here's a <a href="http://freecodecamp.org"> link to freecodecamp.org</a> for you to follow.</p>

### PR DESCRIPTION
**Pull request by Ryan-ED**
Text and external links. Image was not clear in explaining the
composition of anchor tags.

Closes #16289

#### Pre-Submission Checklist

- [x] Your pull request targets the `staging` branch of freeCodeCamp.
- [x] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [x] You have only one commit (if not, [squash](http://forum.freecodecamp.org/t/how-to-squash-multiple-commits-into-one-with-git/13231) them into one commit).
- [x] All new and existing tests pass the command `npm test`. Use `git commit --amend` to amend any fixes.

#### Type of Change

- [ ] Small bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Add new translation (feature adding new translations)

#### Checklist:

- [x] Tested changes locally.
- [x] Addressed currently open issue (replace XXXXX with an issue no in next line)

Closes #16289

#### Description

The image and description for this challenge were not clear or properly laid out. I simply added a list of instructions that broke down what the image was attempting to explain. I made sure that the text was formatted properly using the right HTML tags and escape characters.

### CODE REVIEW

File: seed/challenges/01-responsive-web-design/basic-html-and-html5.json [hunk 1]
```diff
@@ -708,9 +708,15 @@
...
-        "<a href=\"//i.imgur.com/hviuZwe.png\" target=\"_blank\" data-lightbox=\"img-enlarge\"><img class=\"img-responsive\" title=\"Click to enlarge\" alt=\"a diagram of how anchor tags are composed with the same text as on the following line\" src=\"//i.imgur.com/hviuZwe.png\"></a>",
+        "<a href=\"//i.imgur.com/hviuZwe.png\" target=\"_blank\" data-lightbox=\"img-enlarge\"><img class=\"img-responsive\" title=\"Click to enlarge\" alt=\"a diagram of how anchor tags are composed with the same text as on the following line\" src=\"//i.imgur.com/hviuZwe.png\">Click to enlarge </a>",
         "Here's an example:",
-        "<code>&#60;p&#62;Here's a &#60;a href=\"http://freecodecamp.org\"&#62; link to freecodecamp.org&#60;/a&#62; for you to follow.&#60;/p&#62;</code>",
+        "<blockquote>&#60;p&#62;<br>  Here's a <br>  &#60;a href=\"http://freecodecamp.org\"&#62; <br>    link to freecodecamp.org<br>  &#60;/a&#62;<br>  for you to follow.<br>&#60;/p&#62;</blockquote>",
+        "Let's break down the example:",
+        "1. Normal text in the p element<br>  <code>&#60;p&#62;Here's a ... for you to follow.&#60;/p&#62;</code>",
```
Comment (raisedadead) [inline seed/challenges/01-responsive-web-design/basic-html-and-html5.json]:
```diff
-
        "1. Normal text in the p element<br>  <code>&#60;p&#62;Here's a ... for you to follow.&#60;/p&#62;</code>",
        "2. Anchor tag (requires a closing tag)<br>  <code>&#60;a&#62;&#60;/a&#62;</code>",
        "3. <code>href</code> is an anchor tag attribute<br>  <code>&#60;href=\"http://freecodecamp.org\">&#60;/a&#62;</code>",
        "4. The text within the anchor tags, \"link to freecodecamp.org\", will display a link to click<br>  <code>&#60;p&#62Here's a &#60;a href=\"http://freecodecamp.org\"&#62;link to freecodecamp.org&#60;/a&#62;&#60;/p&#62;</code>",
        "5. Final render:<br><p>Here's a <a href=\"http://freecodecamp.org\"> link to freecodecamp.org</a></p>",
+
        "Normal text is wrapped in the <code>p</code> element:<br> <code>&#60;p&#62; Here's a ... for you to follow. &#60;/p&#62;</code>",
        "Next is the <code>anchor</code> element <code>&#60;a&#62;</code> (which requires a closing tag <code>&#60;/a&#62;</code>):<br>  <code>&#60;a&#62; ... &#60;/a&#62;</code>",
        "<code>href</code> is an anchor tag attribute that contains the URL address of the link:<br>  <code>&#60;a href=\"http://freecodecamp.org\"> ... &#60;/a&#62;</code>",
        "The text, <strong>\"link to freecodecamp.org\"</strong>, within the anchor element called <code>anchor text</code>, will display a link to click:<br>  <code>&#60;a href=\" ... \"&#62;link to freecodecamp.org&#60;/a&#62;</code>",
        "The final output of the example will look like this:<br><p>Here's a <a href=\"http://freecodecamp.org\"> link to freecodecamp.org</a></p>",
```

**Review by raisedadead — CHANGES_REQUESTED** [review]
A few nit-picks (mostly styling and verbiage suggestions).

**Comment by raisedadead** [comment]
Hi @Ryan-ED nice work and thanks for the PR.

We have few nit picking to do (trust me its a good thing, when the change is too good to find faults with, 😛!)

I have added them in the review comments, above.
- Lets remove the numbered points, they look simple without them.
- I have added a few spaces, and `...` here and there to enhance the readability.
- Also, some suggestions to the verbiage, to simplify the intent of the instructions.

And now, understanding that you are doing this for first time here is how you would want to do it:

```sh
git checkout fix/link-to-external-pages
# Make changes to the file as suggested above
git add seed/challenges/01-responsive-web-design/basic-html-and-html5.json
git commit --amend
```
Edit the commit message title:

> **style(seed\challenges\01-responsive-web-design): Improved instructions. Link to External Pages with** 

=>
 
> **fix(challenges): improved instructions for anchor element challenge** 

The later is smaller, and concise. You can use the description for more info if needed.

```
git push origin --force
```

Also, edit the PR title on this page to the smaller title.

Let us know if you need any assistance in the [Contributors Chat room](https://gitter.im/FreeCodeCamp/Contributors)

**Comment by Ryan-ED** [comment]
Thanks for the feedback @raisedadead . I'm sorting the issue right now!

**Review by raisedadead — APPROVED** [review]

**Comment by raisedadead** [comment]
#### :sparkles: :sparkles: **AWESOME! LGTM.** :sparkles: :sparkles: 

Thanks @Ryan-ED , for this pull request and contributing to the code-base for the first time.
We will be looking forward for more contributions.

Cheers and happy contributing!

**Comment by Ryan-ED** [comment]
Awesome stuff! I'm officially a contributor to open source 😄 . Thanks for all the help @raisedadead .

**Pull request merged**